src/ontology/cl-edit.owl

```diff
@@ -6722,29 +6725,36 @@ SubClassOf(obo:CL_0000393 obo:CL_0000211)
 
 # Class: obo:CL_0000394 (plasmatocyte)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://doi.org/10.1016/B978-012369493-5.50008-0") obo:IAO_0000115 obo:CL_0000394 "A phagocytic hemocyte, responsible for the engulfment of small particles, microbes, and apoptotic tissue debris. It may also secretes antimicrobial peptides and contribute to the production and secretion of proteins of the hemolymph.")
 AnnotationAssertion(rdfs:label obo:CL_0000394 "plasmatocyte")
 SubClassOf(obo:CL_0000394 obo:CL_0000387)
+SubClassOf(obo:CL_0000394 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0035010))
 
 # Class: obo:CL_0000395 (procrystal cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/B978-012369493-5.50008-0") obo:IAO_0000115 obo:CL_0000395 "A precursor of mature crystal cells.")
 AnnotationAssertion(rdfs:label obo:CL_0000395 "procrystal cell")
-SubClassOf(obo:CL_0000395 obo:CL_0000387)
+SubClassOf(obo:CL_0000395 obo:CL_0000385)
 
 # Class: obo:CL_0000396 (lamellocyte)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:https://doi.org/10.1016/B978-012369493-5.50008-0") obo:IAO_0000115 obo:CL_0000396 "A hemocyte found in immuno-stimulated larvae.")
```

Comment:
Text definition missing differential features noted in mapped FBbt term. 
dbxref behind a paywall.

Reply:
> Text definition missing differential features noted in mapped FBbt term.

That’s on purpose. The FBbt definition is rich because lamellocytes are well described in fruit flies. I can’t say for sure that everything we know about _Drosophila melanogaster_ is true of equivalent cells across insects. The CL terms are mostly intended to act as “attach points” where taxon-specific terms (such as the FBbt ones) can be inserted, when building `composite-metazoan` and similar cross-species products.